client/my-sites/domains/components/domain-warnings/test/index.js

```diff
@@ -364,21 +371,25 @@ describe( 'index', () => {
 
 	describe( 'Mutations', () => {
 		test( 'should not mutate domain objects', () => {
+			const expiry = new Date( '2000-09-09' ).toISOString();
+			const registrationDate = new Date( '1999-09-09' ).toISOString();
+
 			const props = {
 				translate: identity,
 				domain: {
 					name: '1.com',
-					registrationMoment: moment( '1999-09-09', 'YYYY-MM-DD' ),
-					expirationMoment: moment( '2000-09-09', 'YYYY-MM-DD' ),
+					registrationDate,
+					expiry,
 				},
 				selectedSite: { domain: '1.com' },
+				moment,
 			};
 
 			TestUtils.renderIntoDocument( <DomainWarnings { ...props } /> );
 
-			expect( props.domain.name ).to.equal( '1.com' );
-			assert( props.domain.registrationMoment.isSame( moment( '1999-09-09', 'YYYY-MM-DD' ) ) );
-			assert( props.domain.expirationMoment.isSame( moment( '2000-09-09', 'YYYY-MM-DD' ) ) );
+			expect( props.domain.name ).toBe( '1.com' );
+			expect( props.domain.registrationDate ).toEqual( registrationDate );
+			expect( props.domain.expiry ).toEqual( expiry );
```

Comment:
I know this is a preexisting test, but it doesn't make any sense to me what it tries to test. Pass props to a component and then verify that the component indeed got the props? 🤔 Sounds tautological.

Reply:
It's testing for things not to be mutated, but I have no idea how that would happen 🤷‍♂ Removed.